bin/openstack_taster

```diff
@@ -12,8 +12,8 @@ settings = {}
 ARGV << '-h' if ARGV.empty?
 
 parser = OptionParser.new do |opts|
-  opts.banner = "Usage: openstack_taster <image_name> {suite_name} [--create-snapshot]\n" \
-                "   or: openstack_taster <image_name> [--create-snapshot]\n" \
+  opts.banner = "Usage: openstack_taster <image_name> <ssh_user> {suite_name}  [--create-snapshot]\n" \
+                "   or: openstack_taster <image_name> <ssh_user> [--create-snapshot]\n" \
                 '   or: openstack_taster <option>'
```

Comment:
What is `openstack_taster <option>` for? Below it appears you removed the case where `params.length == 1`, suggesting this may no longer be a valid usage?

Reply:
The only valid option is the `-h` option, which causes the program to quit with a usage message before the case statement.